centralserver/i18n/management/commands/update_pot.py

```diff
@@ -1,166 +0,0 @@
-"""
```

Comment:
However, retaining these functions makes our lives more complicated, since they use the deprecated i18n functions.